src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days <= 0) {
+    return 'error';
+  }
+
+  if (days < 3) {
+    return days * 40;
```

Comment:
move all magic numbers like 40 .. to the constants

Reply:
Is it really necessary for such a small function(not including 40)?